### ISSUE TITLE
Problem with fonts

### ISSUE THREAD

**Issue by polemius** [onivim/oni2#2378]
Hi, first of all thank you for the wonderful work! I really like the onivim2!

After new release I have some strange behavior of rendering characters:

<img width="342" alt="Screen Shot 2020-08-30 at 19 38 50" src="https://user-images.githubusercontent.com/48512663/91665795-7e4d3700-eaf8-11ea-8992-6038119c32fd.png">

Here is my `configuration.json`:

```json
{
  "workbench.colorTheme": "Dark+ (default dark)",
  "editor.detectIndentation": true,
  "editor.fontFamily": "FiraCode-Regular.ttf",
  "editor.fontSize": 14,
  "editor.largeFileOptimizations": true,
  "editor.highlightActiveIndentGuide": true,
  "editor.indentSize": 4,
  "editor.insertSpaces": true,
  "editor.lineNumbers": "relative",
  "editor.matchBrackets": true,
  "editor.minimap.enabled": true,
  "editor.minimap.maxColumn": 120,
  "editor.minimap.showSlider": true,
  "editor.renderIndentGuides": true,
  "editor.renderWhitespace": "all",
  "editor.rulers": [],
  "editor.tabSize": 4,
  "editor.zenMode.hideTabs": true,
  "editor.zenMode.singleFile": true,
  "files.exclude": [ "_esy", "node_modules", ".git" ],
  "workbench.activityBar.visible": true,
  "workbench.editor.showTabs": false,
  "workbench.iconTheme": "vs-seti",
  "workbench.sideBar.visible": true,
  "workbench.statusBar.visible": true,
  "workbench.tree.indent": 2,
  "vim.useSystemClipboard": [ "yank" ]
}
```  

Maybe it is something wrong with font. How I could restart the configuration.json file to default one?

Thanks

**Comment by CrossR** [comment]
Sorry! We moved from `Fira Code` to our current font `Jet Brains Mono` and thought all the fallback cases had been caught, but apparently not. If you delete the `editor.fontFamily` line in your config, it will use the default font now instead.

(We changed as Fira doesn't have an italic variant I believe, which is a pain when some themes wil render comments etc in italics, wheras Jet Brains Mono does not have that issue. It *should* just check and see "Tried X font and it doesn't exist, so defaulting back to the Oni2 default" but that doesn't seem to have worked here).
